package.json

```diff
@@ -27,7 +28,7 @@
     "three": ">=0.126.1"
   },
   "scripts": {
-    "dev": "rollup --config --watch",
+    "dev": "rollup --config --watch & serve -S -p 3000 examples/ && kill $!",
```

Comment:
`npm run dev` now `rollup --watch` and `serve` the `examples` folder at the same time